core/src/main/scala/org/apache/spark/sql/SnappyDDLParser.scala

```diff
@@ -618,7 +618,12 @@ abstract class SnappyDDLParser(session: SparkSession)
   protected def alterTable: Rule1[LogicalPlan] = rule {
     ALTER ~ TABLE ~ tableIdentifier ~ (
         ADD ~ COLUMN.? ~ column ~ defaultVal ~ EOI ~> AlterTableAddColumnCommand |
-        DROP ~ COLUMN.? ~ identifier ~ EOI ~> AlterTableDropColumnCommand |
+        DROP ~ COLUMN.? ~ identifier ~ EOI ~> ((table: TableIdentifier, col: String)
+        => AlterTableDropColumnCommand(table, col)) |
+        DROP ~ COLUMN.? ~ identifier ~ CASCADE ~ EOI ~> ((table: TableIdentifier, col: String)
+        => AlterTableDropColumnCommand(table, col, "cascade")) |
```

Comment:
Minor suggestion -- Little refactoring to not look repeat code:

`DROP ~ COLUMN.? ~ identifier ~ (CASCADE ~ push("cascade") | (RESTRICT ~ push ("restrict"))) ~ EOI ~> ((table: TableIdentifier, col: String, refaction: String)
=> {
refaction match {
case "cascade" => AlterTableDropColumnCommand(table, col, "cascade")
case _ => AlterTableDropColumnCommand(table, col, "restrict")
}
}) |`